cookie-getter.js

```diff
@@ -7,5 +7,5 @@ module.exports = function (name) {
   var res
   if (!~setPos) return null
   res = decodeURIComponent(cookie.substring(setPos, ~stopPos ? stopPos : undefined).split('=')[1])
-  return (res.charAt(0) === '{') ? JSON.parse(res) : res
+  return (res.charAt(0) === '{') ? JSON.parse(res) : ((res.indexOf('j:{') === 0) ? JSON.parse(res.substr(2)) : res)
```

Comment:
At this point, probably better to restructure this as an `if` block. Nested tertiary functions are a bit hard to reason through.

Thanks for the PR!